src/state/store.ts

```diff
@@ -126,23 +113,17 @@ const stateSanitizer = (state: any) => {
 /// This allows us to create an empty store for tests
 export const createStore = () =>
   configureStore({
-    reducer: persistedReducer,
+    reducer,
     enhancers: existingEnhancers => {
       // Add the autobatch enhancer to the store setup
       return existingEnhancers.concat(autoBatchEnhancer())
     },
     middleware: getDefaultMiddleware =>
       getDefaultMiddleware({
-        immutableCheck: {
-          warnAfter: 128,
-          ignoredActions: [PERSIST, PURGE],
-        },
-        serializableCheck: {
-          ignoreState: true,
-          ignoreActions: true,
-          warnAfter: 128,
-          ignoredActions: [PERSIST, PURGE],
-        },
+        // funnily enough, the checks that should check for perf. issues are actually slowing down the app
+        // https://github.com/reduxjs/redux-toolkit/issues/415
```

Comment:
[suggestion]: Comment why disabling this doesnt reduce safety so we dont have a panik moment later

Reply:
https://github.com/shapeshift/web/pull/7799/commits/8cc1c28b758a5323db86385883f47fed47b88696